packages/lib/src/testing/jest-setupAfterEnv.ts

```diff
@@ -31,8 +31,13 @@ const setupPermanentMocks = () => {
   Math.random = () => TEST_DEFAULT_MATH_RANDOM; // To customize in a given test, use `jest.spyOn(Math, "random").mockReturnValue(xx)`
 };
 
+jest.setTimeout(TEST_TIMEOUT);
```

Comment:
how about putting this only in the problematic file? that would point out the issue more precisely?

(btw, you only need it from the before each I'd say?)

Reply:
Done :)